### PR TITLE
style(experience): update scopes and organizations selector style

### PR DESCRIPTION
**Pull request by simeng-li**
## Summary

Update the scopes and organization selector style.

This PR includes the following update:

1. Remove all the IconButton wrappers.  Expand the clickable area to the entire item element instead of the IconButton. 
2. Make all the list item cursor pointer.
3. Update the organization selector's style to align with other input selectors. Add active border and outline when the dropdown menu is open
4. Adjust the organization dropdown modal spacing on mobile. 

image

image

## Testing

test locally

## Checklist

- [ ] ~`.changeset`~
- [ ] ~unit tests~
- [ ] ~integration tests~
- [ ] ~necessary TSDoc comments~

### CODE REVIEW

File: packages/experience/src/pages/Consent/OrganizationSelector/OrganizationItem/index.tsx [hunk 1]
```diff
@@ -21,27 +23,27 @@ const OrganizationItem = ({
...
-              onSelect(organization);
-            },
-            ' ': () => {
-              onSelect(organization);
-            },
-          });
-        },
+        onKeyDown: onKeyDownHandler({
+          Enter: () => {
+            console.log('enter');
```
Comment (charIeszhao) [inline packages/experience/src/pages/Consent/OrganizationSelector/OrganizationItem/index.tsx:40]: Unexpected?

File: packages/experience/src/pages/Consent/OrganizationSelector/OrganizationItem/index.tsx [hunk 2]
```diff
@@ -21,27 +23,27 @@ const OrganizationItem = ({
...
+            onSelect(organization);
+          },
+          ' ': () => {
+            onSelect(organization);
```
Comment (charIeszhao) [inline packages/experience/src/pages/Consent/OrganizationSelector/OrganizationItem/index.tsx:44]: Also I think the `onKeyDownHandler` deals with space and enter keys by default, if you only pass the callback function.

image

File: packages/experience/src/pages/Consent/OrganizationSelector/OrganizationItem/index.tsx (hunk 1 above)
Comment (simeng-li) [reply packages/experience/src/pages/Consent/OrganizationSelector/OrganizationItem/index.tsx:40]: updated

File: packages/experience/src/pages/Consent/OrganizationSelector/OrganizationItem/index.tsx (hunk 2 above)
Comment (simeng-li) [reply packages/experience/src/pages/Consent/OrganizationSelector/OrganizationItem/index.tsx:44]: cool updated

**Review by charIeszhao — APPROVED** [review]

**Pull request merged**